core/src/io/anuke/mindustry/world/blocks/storage/Unloader.java

```diff
@@ -50,6 +52,7 @@ public void playerPlaced(Tile tile){
     public void configured(Tile tile, Player player, int value){
         tile.entity.items.clear();
         tile.<UnloaderEntity>entity().sortItem = content.item(value);
+        playerLog.record(player, PlayerLog.Action.configured, tile, null);
```

Comment:
This should be recorded after each configure at the end of `InputHandler#onTileConfig`, not in each block type.